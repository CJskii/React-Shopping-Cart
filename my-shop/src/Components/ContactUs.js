import React from "react";

const ContactUs = () => {
  return <div className="text-white text-center text-5xl">Contact Us</div>;
};

export default ContactUs;
