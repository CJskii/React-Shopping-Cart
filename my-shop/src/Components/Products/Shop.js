import React from "react";
import FeaturedItems from "./Featured";
import Categories from "./Categories";

const Shop = () => {
  return (
    <div className="hero min-h-screen bg-base-200 flex justify-center items-start gap-4">
      {/* <Categories />
      <FeaturedItems /> */}
      <section>
        <div className="max-w-screen-xl px-4 py-8 mx-auto sm:px-6 sm:py-12 lg:px-8">
          <header className="text-center">
            <h2 className="text-xl font-bold text-base-content sm:text-3xl">
              New Arrivals
            </h2>

            <p className="max-w-md mx-auto mt-4 text-gray-500">
              Explore our range of high power ASIC miners, graphic cards and
              cool crypto gadgets!
            </p>
          </header>

          <ul className="grid grid-cols-1 gap-4 mt-8 lg:grid-cols-3">
            <li>
              <a href="#" className="relative block group">
                <img
                  src="/images/gpucat.jpg"
                  alt=""
                  className="object-cover w-full transition duration-500 aspect-square group-hover:opacity-90"
                  style={{ objectFit: "cover" }}
                />

                <div className="absolute inset-0 flex flex-col items-start justify-end p-6">
                  <h3 className="text-xl font-medium text-white bg-black p-2">
                    Graphic Cards
                  </h3>

                  <span className="mt-1.5 inline-block bg-black px-5 py-3 text-xs font-medium uppercase tracking-wide text-white">
                    Shop Now
                  </span>
                </div>
              </a>
            </li>

            <li>
              <a href="#" className="relative block group">
                <img
                  src="/images/asiccat.png"
                  alt=""
                  className="object-cover w-full transition duration-500 aspect-square group-hover:opacity-90"
                />

                <div className="absolute inset-0 flex flex-col items-start justify-end p-6">
                  <h3 className="text-xl font-medium text-white bg-black p-2">
                    Crypto Gadgets
                  </h3>

                  <span className="mt-1.5 inline-block bg-black px-5 py-3 text-xs font-medium uppercase tracking-wide text-white">
                    Shop Now
                  </span>
                </div>
              </a>
            </li>

            <li className="lg:col-span-2 lg:col-start-2 lg:row-span-2 lg:row-start-1">
              <a href="#" className="relative block group">
                <img
                  src="/images/btc-mining.png"
                  alt=""
                  className="object-cover w-full transition duration-500 aspect-square group-hover:opacity-90"
                />

                <div className="absolute inset-0 flex flex-col items-start justify-end p-6">
                  <h3 className="text-xl font-medium text-white  bg-black p-2 rounded">
                    ASIC miners
                  </h3>

                  <span className="mt-1.5 inline-block bg-black px-5 py-3 text-xs font-medium uppercase tracking-wide text-white">
                    Shop Now
                  </span>
                </div>
              </a>
            </li>
          </ul>
        </div>
      </section>

      {/* <ShopCard />
      <ShopCard />
      <ShopCard />
      <ShopCard /> */}
    </div>
  );
};

export default Shop;
