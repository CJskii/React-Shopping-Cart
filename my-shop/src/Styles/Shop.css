.feature-img {
  object-fit: cover;
}
