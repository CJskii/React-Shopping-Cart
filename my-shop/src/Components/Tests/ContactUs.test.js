import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import userEvent from "@testing-library/user-event";
import Shop from "../Products/Shop";
import { MemoryRouter } from "react-router-dom";
import ContactUs from "../ContactUs";

test("renders contact us page", () => {
  const { fragment } = render(
    <MemoryRouter>
      <ContactUs />
    </MemoryRouter>
  );

  expect(fragment).toMatchSnapshot();
});
