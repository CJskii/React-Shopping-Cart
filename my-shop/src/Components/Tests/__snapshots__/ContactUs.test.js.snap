// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders contact us page 1`] = `undefined`;
